src/app/content/studyGuides/hooks/printStudyGuides.spec.ts

```diff
@@ -86,9 +107,25 @@ describe('printStudyGuides', () => {
     expect(calmSpy).toHaveBeenCalled();
     await Promise.resolve();
 
+    expect(dispatch).toBeCalledWith(toggleStudyGuidesSummaryLoading(false));
     expect(print).toHaveBeenCalled();
   });
 
+  it('doesn\'t wait for promiseCollector.calm', async() => {
```

Comment:
is this required for coverage for some if path?

Reply:
Yup. It's about `  if (waitingForPromiseCollector) {...}` else path.
Maybe the test name is misleading or smthing.